### ISSUE TITLE
clearer titles and abstracts

### ISSUE THREAD

**Issue by fredgibbs** [programminghistorian/jekyll#406]
As we continue site standardization and layout work, I think it would be worth rephrasing overly wordy title and abstracts so that the lessons page and lessons headers look more uniform and make the lessons a bit more inviting (and clear as to their essence). Some of the lesson titles and abstracts (including many I edited) are far too wordy and off-putting. We can do this without breaking URLs, I think, using permalinks in the YAML, but I wanted to gather feedback here and flag any potential problems. it's something I wanted to put on the table now since we have related metadata improvement work underway, and I've been clicking through almost all lessons on a regular basis.

**Comment by amandavisconti** [comment]
This sounds good to me. When revising the front matter for all lessons in my /proghistdev fork, I left all titles as-is, but I added abstracts for any lessons that were missing one—so we could either work from those, or I could not push that part of my code.

**Comment by acrymble** [comment]
I believe 'abstracts' need to become part of the editorial workflow. Do we want authors, editors, or someone else in charge of that?

**Comment by ianmilligan1** [comment]
I think abstracts should be treated as part of the lesson - so authors responsible with editorial oversight.

**Comment by fredgibbs** [comment]
it seems the last 5 or so lessons published during our redesign work do not have abstracts, which makes them look incomplete on the new lessons index page.
![screen shot 2017-07-11 at 11 05 23 am](https://user-images.githubusercontent.com/515448/28080556-a930199e-6629-11e7-9344-e7d36aca1119.png)

**Comment by mdlincoln** [comment]
I'm going to write up short abstracts for these five lessons, so that we can launch the redesign. We can always go back in and change them later.
